### PR TITLE
Add TCK for JSON schema validation

### PR DESCRIPTION
**Pull request by xstefank**
Signed-off-by: xstefank <xstefank122@gmail.com>

### CODE REVIEW

**Comment by xstefank** [comment]
This a same PR as #175. 

@antoinesd can you file the appropriate CQ, please?

File: tck/pom.xml [hunk 1]
```diff
@@ -112,6 +112,12 @@
             <version>${version.glassfish.json}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.java-json-tools</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>2.2.10</version>
+        </dependency>
```
Comment (antoinesd) [inline tck/pom.xml]: Did you try to exclude some of the dependencies here to limit extra lib and CQ ?

Comment (xstefank) [reply tck/pom.xml]: no, but I can try, is there something I can help you with the filing of the CQ?

**Comment by xstefank** [comment]
@antoinesd thanks! But are we clear on the CQs this time, please?

**Comment by antoinesd** [comment]
It's taken into account in #200

**Comment by xstefank** [comment]
@antoinesd thank you!

**Pull request merged**